src/client/pages/Checkout/CheckoutDetails/components/YourPlan/YourPlan.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { getFormattedPrice } from 'utils/getFormattedPrice'
+import { usePriceQuery } from 'data/graphql'
+import { Badge } from 'components/Badge/Badge'
+import { MEDIUM_SMALL_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { SubSection } from '../SubSection'
+
+const { gray700, gray300 } = colorsV3
+
+export const Row = styled.div`
+  font-size: 0.875rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0.5rem 0;
+  line-height: 1.4;
+`
+
+const Value = styled.div`
+  color: ${gray700};
+`
+const Label = styled.div``
+
+const Total = styled.div`
+  border-top: 1px solid ${gray300};
```

Comment:
Instead of `border-top` I think we should have a `Divider` (or `HorizontalDivider` or `HorizontalRuler`) component that we can reuse, since this horizontal divider appears in a lot of places on the new checkout pages. I know @amandaflod created on but it's in the `QuoteDetails.tsx`. I think it would be great if that could be moved into its own file in the `/Checkout/shared` folder so we can reuse it wherever we need this.